### PR TITLE
Add groupdata for URL Fragment Text Directives spec

### PR DESCRIPTION
**Pull request by Josh-Cena**
Add a missing sidebar

### CODE REVIEW

File: files/en-us/web/api/url_fragment_text_directives/index.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+---
+
+{{DefaultAPISidebar("URL Fragment Text Directives")}}{{SeeCompatTable}}
+
+The **URL fragment text directives** API allows web apps to interact with _text fragments_ in the URL. [Text fragments](/en-US/docs/Web/URI/Fragment/Text_fragments) allow linking directly to a specific portion of text in a web document, without requiring the author to annotate it with an ID, using a particular syntax in the URL fragment.
+
+## Interfaces
+
+- {{domxref("FragmentDirective")}}
+  - : An object that's empty at the moment.
```
Comment (hamishwillee) [inline files/en-us/web/api/url_fragment_text_directives/index.md:19]:
```suggestion
  - : An empty object that indicates whether or not the browser supports text fragments.
```

Comment (hamishwillee) [reply files/en-us/web/api/url_fragment_text_directives/index.md:19]: ^^^ We can add additional description or change this when/if it used for something else.

Comment (Josh-Cena) [reply files/en-us/web/api/url_fragment_text_directives/index.md:19]: My problem with "indicates whether or not the browser supports text fragments" (which is also the wording used on `FragmentDirective` itself) is that this interface itself cannot feature-detect. Rather, the property `document.fragmentDirective` is used to feature-detect. The interface is actually supposed to contain relevant information, of which there's none.

Comment (hamishwillee) [reply files/en-us/web/api/url_fragment_text_directives/index.md:19]: YOu make a fair point, but what's here requires me to jump around for answers about the point of this interface "at the moment". Would this be better?

```suggestion
  - : A (currently) empty object.
     The existence of an object of this type in {{domxref("document.fragmentDirective")}} is used for feature detection.
```

**Review by hamishwillee — APPROVED** [review]
Thanks very much @Josh-Cena 

Approving so you can merge, or wait for other guys to further sanity check.

**Comment by Josh-Cena** [comment]
I intended for this to be a simple IA fix and later realized that a sidebar required an overview so I wrote a very brief one. Thanks for the review!

**Pull request merged**